Source/Scene/QuadtreePrimitive.js

```diff
@@ -182,6 +187,14 @@ define([
         }
     };
 
+    QuadtreePrimitive.prototype.addTileLoadedCallback = function(callback) {
```

Comment:
In general, the control flow seems more complicate than I imagined.  I dunno...perhaps it needs to be.  You'll have to walk me through it tomorrow.